File: Music.Short.Android/app/src/main/java/com/zitherharpmusic/zhmshort/ui/empty/EmptyFragment.java
package com.zitherharpmusic.zhmshort.ui.empty;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import com.zitherharpmusic.zhmshort.R;

public class EmptyFragment extends Fragment {
    private static String text;

    public static EmptyFragment newInstance(String text) {
        EmptyFragment.text = text;
        return new EmptyFragment();
    }

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater,
                             @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_empty, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        TextView textView = view.findViewById(R.id.empty);
        textView.setText(text);
    }
}

File: Music.Client.Android/Kotlin/Store/app/src/main/java/com/zitherharp/store/ui/empty/EmptyFragment.kt
package com.zitherharp.store.ui.empty

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.zitherharp.store.databinding.FragmentEmptyBinding

class EmptyFragment(private val text: String) : Fragment() {

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?): View {
        return FragmentEmptyBinding.inflate(inflater, container, false).apply {
            status.text = text
        }.root
    }
}